CONFIGDOC.md

```diff
@@ -428,40 +441,163 @@ pipeline {
+- Multi-configuration projects — In the **Configuration Matrix** section, add the **MATLAB** axis and then select your preferred versions. For an example, see [Add MATLAB Axis](#add-matlab-axis).
+
+To use a tool configured using MATLAB Package Manager in a freestyle or multi-configuration project, you must also associate the tool with a valid license in your project. If you have a MATLAB batch licensing token, you can address this requirement by setting the `MLM_LICENSE_TOKEN` environment variable in the **Environment** section of the project configuration window. For example, suppose that: 
+
+- A tool named `Latest` automatically installs the latest release of MATLAB on your agent.
+- A secret-text credential with `MATLAB batch licensing token` as its name secures access to your token. (For how to create a credential from a batch licensing token, see [License Installed Products](#license-installed-products).) 
+
+To use the tool named `Latest` in a freestyle project, configure the **Environment** section by binding the credential to the  `MLM_LICENSE_TOKEN` environment variable and specifying the MATLAB version to use for the build:
+
+- To bind the credential, select **Use secret text(s) or file(s)**, enter `MLM_LICENSE_TOKEN` in the **Variable** box and select the credential from the **Credentials** list.
+- To specify the MATLAB version, select **Use MATLAB version** and then select `Latest` from the list.
+
+![Environment section showing how to bind a credential to the MLM_LICENSE_TOKEN environment variable and how to select the latest release of MATLAB for the build](https://github.com/user-attachments/assets/749f5ae9-a105-4481-bf60-19c136ee1447)
+
+For more information about freestyle and multi-configuration projects, see [Set Up Freestyle Project](#set-up-freestyle-project) and [Set Up Multi-Configuration Project](#set-up-multi-configuration-project).
+
 ### Use MATLAB as a Tool in Pipeline Project
-To invoke MATLAB as a Jenkins tool using declarative pipeline syntax, use a `tools` block in your `Jenkinsfile`. To specify the tool in the block, use the `matlab` keyword followed by the name assigned to the tool on the **Tools** page. For example, run `myscript.m` using the MATLAB version that has been registered as a tool named R2024a.  
+To invoke MATLAB as a Jenkins tool using declarative pipeline syntax, use a `tools` block in your `Jenkinsfile`. To specify the tool in the block, use the `matlab` keyword followed by the name assigned to the tool on the **Tools** page. For example, run `myscript.m` using a preinstalled MATLAB version that has been registered as a tool named `R2024b`.  
 
 ```groovy
 // Declarative Pipeline
 pipeline {
    agent any
    tools {
-       matlab 'R2024a'
+       matlab 'R2024b'
    }
     stages {
         stage('Run MATLAB Command') {

```

Comment:
Should we also show how to use a token in the Declarative Pipeline? I feel like this example is more important than the Freestyle one.

Reply:
Edit: Nevermind, I see you have it even further below. It just confused me for a minute.